helpers/deploy/deploymentUtils.ts

```diff
@@ -1,9 +1,11 @@
 import { ethers, getNamedAccounts } from "hardhat";
 
-import bscMainnetGovernanceDeployments from "../../deployments/bscmainnet.json";
 import bscTestnetGovernanceDeployments from "../../deployments/bsctestnet.json";
 import { SUPPORTED_NETWORKS } from "./constants";
 
+export const testnetNetworks = ["sepolia", "opbnbtestnet", "arbitrumsepolia"];
+const mainnetNetworks = ["ethereum", "opbnbmainnet", "arbitrumone"];
```

Comment:
```suggestion
const mainnetNetworks = ["ethereum", "opbnbmainnet", "arbitrumone", "hardhat"];
```

Is the quickest way to test this in development to deploy both sides of governance to hardhat? My thought is to start this way for testing/ subgraph testing and then iterate.

Reply:
Scripts are working fine on hardhat, we don't need this change.

Reply:
Correct the work fine but what we do loose without this change is deploying the target network contracts to hardhat which could be helpful for testing for example.

Reply:
We can still deploy target network contracts along with local network contracts on hardhat .
For hardhat it will take bscmainnet chain id by default, see [here](https://github.com/VenusProtocol/governance-contracts/pull/57/files#diff-d99dff738cce4277839e2cd5eeba92e0a3c2b2dfbe8fce3868972ae0748c918dR84) 
LMK if I am missing something.

Reply:
[df92411](https://github.com/VenusProtocol/governance-contracts/pull/57/commits/df924118648dcc726c2966531ba7ebe1d973bd33)